xml/deploy_cephadm.xml

```diff
@@ -652,8 +652,12 @@ o- time_server ................................................ [enabled]
      Optionally, you can set a local container registry. This will serve as a
      mirror of the <literal>registry.suse.com</literal> registry. Remember you
      need to re-sync the local registry whenever there are new updated
-     containers available from <literal>registry.suse.com</literal>. Creating a
-     local registry is useful in the following scenarios:
+     containers available from <literal>registry.suse.com</literal>.  Find more
```

Comment:
Is &lt;literal&gt; the most appropriate tag for this?

How about 
&lt;systemitem class="systemname"&gt;registry.suse.com&lt;/systemitem&gt;
?